### PR TITLE
Test on Julia 1.4

### PR DESCRIPTION
**Pull request by sostock**
This adds Julia 1.4 to the CI matrix.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ environment:
   - julia_version: 1.1
   - julia_version: 1.2
   - julia_version: 1.3
```
Comment (giordano) [inline appveyor.yml:6]: I'd personally remove these versions, they're unsupported.  Also, the line above should be `1.0`, not `1`, we're testing v1.4 twice.

File: .travis.yml [hunk 2]
```diff
@@ -8,6 +8,7 @@ julia:
   - 1.1
   - 1.2
   - 1.3
```
Comment (giordano) [inline .travis.yml:10]: Same as the AppVeyor, I'd remove all these versions

File: appveyor.yml (hunk 1 above)
Comment (sostock) [reply appveyor.yml]: OK, I removed them. We are now testing on `1.0` and `1`, so this (hopefully) does not have to be updated until the next Julia LTS release.

File: .travis.yml (hunk 2 above)
Comment (sostock) [reply .travis.yml]: Done :)

**Pull request merged**